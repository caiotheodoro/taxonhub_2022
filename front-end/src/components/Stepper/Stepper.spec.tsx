import { render } from "@testing-library/react";
import { Stepper } from "./index";

describe("Pagination components", () => {
  it("should render correctly", () => {
    const { container } = render(
      <Stepper
      height="100px"
      isLastStep={false}

      >
        .
      </Stepper>
    );
    expect(container).toMatchSnapshot();
  })

});
