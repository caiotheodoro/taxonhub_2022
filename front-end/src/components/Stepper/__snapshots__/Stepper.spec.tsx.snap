// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination components should render correctly 1`] = `
<div>
  <div
    style="opacity: 0;"
  >
    <div
      class="css-1xhj18k"
    >
      <div
        class="css-cih3bh"
      >
        <div
          class="css-swgfv8"
        />
        <div
          class="css-1jfxfpp"
        />
      </div>
      .
    </div>
  </div>
</div>
`;
